mqtt-streaming/src/main/scala/akka/stream/alpakka/mqtt/streaming/MqttSessionSettings.scala

```diff
@@ -182,14 +181,6 @@ final class MqttSessionSettings private (val maxPacketSize: Int = 4096,
   def withReceiveUnsubAckTimeout(receiveUnsubAckTimeout: Duration): MqttSessionSettings =
     copy(receiveUnsubAckTimeout = receiveUnsubAckTimeout.asScala)
 
-  /**
-   * For clients, the number of commands to buffer in memory while waiting for a connection ack. For servers,
-   * the number of commands to buffer in memory while waiting for a local connection ack command. Defaults
-   * to 100.
-   */
-  def withMaxClientConnectionStashSize(maxClientConnectionStashSize: Int): MqttSessionSettings =
-    copy(maxClientConnectionStashSize = maxClientConnectionStashSize)
-
```

Comment:
Better to let the call control the number of messages being sent rather than introducing an arbitrary limit.